focuspoints.lrdevplugin/DefaultPointRenderer.lua

```diff
@@ -42,15 +43,61 @@ function DefaultPointRenderer.createView(photo, photoDisplayWidth, photoDisplayH
   local developSettings = photo:getDevelopSettings()
   local metaData = ExifUtils.readMetaDataAsTable(photo)
 
+  local lrRotation, lrMirroring = DefaultPointRenderer.getLightroomRotationAndMirroring(photo)
```

Comment:
avoid prefixing with lr or Lr as the docs point out. Lightroom reserves this. I dont trust LR enough to not conflict on case.